src/pages/YearPickerPage.js

```diff
@@ -64,7 +64,7 @@ class YearPickerPage extends React.Component {
      */
     updateSelectedYear(selectedYear) {
         // We have to navigate using concatenation here as it is not possible to pass a function as a route param
-        Navigation.navigate(`${this.props.route.params.backTo}?year=${selectedYear}`);
+        Navigation.goBack(`${ROUTES.SETTINGS_PERSONAL_DETAILS_DATE_OF_BIRTH}?year=${selectedYear}`, true);
```

Comment:
Ah so we hard coded the route, but was the original intention for this to be more generic? This change would break the re-usability of this component I think...